node/middlewares/customRoutes.ts

```diff
@@ -0,0 +1,52 @@
+import { getAppsRoutes, getUserRoutes } from '../services/routes'
+
```

Comment:
```suggestion
const metricsConfig = {
  customAppRoutes: {
    failed: 'customRoutes-getAppsRoutes-failed',
    success: 'customRoutes-getAppsRoutes-success',
  },
  customUserRoutes: {
    failed: 'customRoutes-getUserRoutes-failed',
    success: 'customRoutes-getUserRoutes-success',
  },
  customRoutes: {
    success: 'customRoutes-success',
  },
};

```

Addin as a object so we can manage better if we want to add few more metrics or for scalability in different places